jetty-server/src/main/java/org/eclipse/jetty/server/session/FileSessionDataStore.java

```diff
@@ -165,7 +165,7 @@ public boolean deleteFile (String filename) throws Exception
     public Set<String> doGetExpired(final Set<String> candidates)
     {
         final long now = System.currentTimeMillis();
-        HashSet<String> expired = new HashSet<String>();
+        Set<String> expired = new HashSet<>();
```

Comment:
Lets just stick to hazelcast for this PR. By all means just make that small change directly to the FileSessionDataStore in a separate commit.

Reply:
oh yes sorry I was looking at this class and my intelligent ide was telling me this is not needed :P